tensorflow-core/tensorflow-core-generator/src/main/java/org/tensorflow/processor/operator/OperatorProcessor.java

```diff
@@ -562,25 +562,7 @@ private static TypeSpec buildTopClass(OpsSpec spec) {
             .returns(Names.Ops)
             .addStatement("return new $T(scope.withInitScope())", Names.Ops)
             .addJavadoc(
-                "Returns an API that builds init operations.  {@link #liftToInitScope(Operand)} will be called for all created operations.\n"
-                    + initScopeComment
-                    + "\n@see #liftToInitScope(Operand)")
-            .build());
-
-    TypeVariableName T = TypeVariableName.get("T").withBounds(Names.Operand);
-    opsBuilder.addMethod(
-        MethodSpec.methodBuilder("liftToInitScope")
-            .addTypeVariable(T)
-            .addModifiers(Modifier.PUBLIC)
-            .addParameter(T, "op")
-            .returns(T)
-            .addStatement("scope.env().registerInitOp(op.op())")
-            .addStatement("return op")
-            .addJavadoc(
-                "Make {@code op} an init operation, doing the same for all of it's inputs (and control inputs).\n"
-                    + initScopeComment
-                    + "\n@see ExecutionEnvironment#registerInitOp(Operation)\n"
-                    + "\n@throws IllegalStateException if the op or one of its inputs can't be made an init op.")
```

Comment:
@rnett I took the liberty of removing this endpoint, assuming that if a user is building a graph and wants an op to be added as a initializer, then he should simply use the `withInitScope()` when creating this op. Wdyt?